.github/workflows/require-pr-label.yml

```
@@ -9,15 +9,50 @@ permissions:
   pull-requests: write
 
 jobs:
-  label:
-    name: DO-NOT-MERGE / unresolved review
+  label-dnm:
+    name: DO-NOT-MERGE
     if: github.repository_owner == 'python'
     runs-on: ubuntu-latest
     timeout-minutes: 10
 
     steps:
-      - uses: mheap/github-action-required-labels@v5
+      - name: Check there's no DO-NOT-MERGE
+        uses: mheap/github-action-required-labels@v5
         with:
           mode: exactly
           count: 0
-          labels: "DO-NOT-MERGE, awaiting changes, awaiting change review"
+          labels: |
+            DO-NOT-MERGE
+  label-reviews:
+    name: Unresolved review
+    if: github.repository_owner == 'python'
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+
+    steps:
+      - name: Check there's no required changes
+        uses: mheap/github-action-required-labels@v5
+        with:
+          mode: exactly
+          count: 0
+          labels: |
+            awaiting changes
+            awaiting change review
+      - id: is-feature
+        name: Check for type-feature
+        uses: mheap/github-action-required-labels@v5
+        with:
+          mode: exactly
+          count: 1
+          labels: |
+            type-feature
+          exit_type: success
+      - id: awaiting-merge
+        if: steps.is-feature.outputs.status == 'success'
+        name: Check for complete review
+        uses: mheap/github-action-required-labels@v5
+        with:
+          mode: exactly
+          count: 1
```

Will this fail if there's two reviews?

Maybe this?

```suggestion
          mode: minimum
          count: 1
```

https://github.com/mheap/github-action-required-labels#usage

---

There is no difference for us here.

`minimum` means you have an "OR" relationship in the list of labels and you expect at least the `count` amount. Since there is only one element in the list of labels you pointed out, minimum 1 is the same as exactly 1.

Labels don't stack on GitHub, they are binary. Either it's there or it isn't. Multiple reviews will not add multiple labels.

---

Ah yes, I think I was mixing up another action that required multiple reviews. This is a different thing here 👍